examples/generalized_linear_models/GLM-robust.ipynb

```diff
@@ -16,7 +16,7 @@
     "\n",
```

Comment:
<p>the StudentT distribution is currently available with the development version of bambi so I thought I should wait till the next stable release and will update this again. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>same doubt about using arviz to plot instead of <code>plot_posterior_predictive_glm()</code> and get rid of the DeprecationWarning as mentioned before.</p>

Reply:
<p>The same code used above should work here. </p><p> </p><pre class="ql-syntax" spellcheck="false">model_robust = bmb.Model("y ~ x", data, family="t")
idata_robust = model_robust.fit(draws=2000, cores=2)

model_robust.predict(idata_robust, data=new_data)


posterior_y_mean = idata_robust.posterior["y_mean"].values.reshape((2 * 2000, 100))

# Create the plot
plt.figure(figsize=(7, 5))


# The [:, :100] selects the 100 samples in your original plot
plt.plot(new_data["x"], posterior_y_mean.T[:, :100], color="k", alpha=0.3, lw=0.5)


plt.plot(x_out, y_out, "x", label="data")

plt.plot(x, true_regression_line, label="true regression line", lw=3.0, c="y")

plt.legend(loc=0);
</pre>

Reply:
<p>I think this is also a good instance to show how to override default priors. You could do something like the following, and then as usual</p><p> </p><pre class="ql-syntax" spellcheck="false"># Use a Gamma prior for the degrees of freedom
model_robust = bmb.Model("y ~ x", data, family="t")
model_robust.set_priors({"nu": bmb.Prior("Gamma", alpha=3, beta=1)})
model_robust
</pre>